README.md

```diff
@@ -24,78 +27,91 @@ A Simple Terminal-Based Mod Organizer for Linux
   - Even minor mistakes are catastrophic.
   - Returning to vanilla requires nuke/pave/reinstall.
 
-# Dependencies
+## Dependencies
+
 - Games installed through Steam
 - Steam is either distro version or from Flatpak com.valvesoftware.Steam
 - Python3
 - p7z (or something else that puts 7z in your PATH).
 
-# Installation Instructions
+## Installation Instructions
+
 Steam Deck users:
-```
+
+```sh
 python -m ensurepip --upgrade
 python -m pip install --upgrade pip
 ```
 
 Everyone:
-```
+
+```sh
 echo 'PATH="$HOME/.local/bin:$PATH"' >> ~/.bashrc
 PATH="$HOME/.local/bin:$PATH"
 git clone https://github.com/cyberrumor/ammo
 cd ammo
 pip3 install -r requirements.txt || pip3 install --break-system-packages -r requirements.txt
 pip3 install . || pip3 install --break-system-packages .
 ```
+
 You can now execute ammo with the terminal command `ammo`.
 
-# Updating Instructions
-```
+## Updating Instructions
+
+```sh
 cd /path/to/ammo/clone/dir
 git pull
 pip3 install --force-reinstall . || pip3 install --break-system-packages --force-reinstall .
 ```
 
-# Usage Instructions
+## Usage Instructions
 
 `ammo` - Launch the interactive shell. Select a game via index if prompted.
 
-```
-activate   mod|plugin <index>             Enabled components will be loaded by game.
-commit                                    Apply pending changes.
-configure  <index>                        Configure a fomod.
-deactivate mod|plugin <index>             Disabled components will not be loaded by game.
-delete     mod|download <index>           Removes specified file from the filesystem.
-exit                                      Quit.
-find       [<keyword> ... ]               Fuzzy filter. 'find' without args removes filter.
-help                                      Show this menu.
-install    <index>                        Extract and manage an archive from ~/Downloads.
-move       mod|plugin <index> <new_index> Larger numbers win file conflicts.
-refresh                                   Abandon pending changes.
-rename     mod|download <index> <name>    Names may contain alphanumerics and underscores.
-```
-
-# Tips and Tricks
+| Command     | Arguments                           | Description |
+|-|-|-|
+| activate    | mod/plugin `<index>`                | Enabled components will be loaded by game |
```

Comment:
We need to preserve the pipes that are used to separate commands. I'm not sure if they need to be escaped, but / has different meaning than | for syntax. I think we may want to remove the code blocks too, they make the args that have them look odd next to the args that do.

The main thing I care about here is that we preserve the standard [docopt](http://docopt.org/) style hints (ignore the fact that I'm missing parenthesis around the pipe operators, just barely became aware I was supposed to have them. I'll fix that later), which is why the pipes matter.

Reply:
Reading up on docopt right now

I think you can escape the pipes but then the issue becomes that in plain text it looks really confusing. I only converted to a table because it looks pretty in the github page haha

Let me ponder on if there is a better way to organize this

Reply:
Escape with backslash is possible! It just makes the plaintext look a little wonky:

```
| Command     | Arguments                             | Description |
|-|-|-|
| activate    | mod\|plugin `<index>`                 | Enabled components will be loaded by game |
| commit      |                                       | Apply pending changes |
| configure   | `<index>`                             | Configure a fomod |
| deactivate  | mod\|plugin `<index>`                 | Disabled components will not be loaded by game |
| delete      | mod\|download `<index>`               | Removes specified file from the filesystem |
| exit        |                                       | Quit. Prompts if there are changes |
| find        | [`<keyword>` ...]                     | Show only components with any keyword; `find` without args resets |
| help        |                                       | Show this menu |
| install     | `<index>`                             | Extract and manage an archive from ~/Downloads |
| move        | mod\|plugin [from_index] [to_index]   | Larger numbers win file conflicts |
| refresh     |                                       | Abandon pending changes |
| vanilla     |                                       | Disable all managed components and clean up |
```

Reply:
```
| Command     | Arguments                             | Description |
|-|-|-|
| activate    | mod\|plugin <index>                   | Enabled components will be loaded by game |
| commit      |                                       | Apply pending changes |
| configure   | <index>                               | Configure a fomod |
| deactivate  | mod\|plugin <index>                   | Disabled components will not be loaded by game |
| delete      | mod\|download <index>                 | Removes specified file from the filesystem |
| exit        |                                       | Quit |
| find        | [<keyword> ...]                       | Show only components with any keyword |
| help        |                                       | Show this menu |
| install     | <index>                               | Extract and manage an archive from ~/Downloads |
| move        | mod\|plugin [from_index] [to_index]   | Larger numbers win file conflicts |
| refresh     |                                       | Abandon pending changes |
| rename      | mod\|download <index> <name>          | Names may contain alphanumerics and underscores |
```
Vanilla command has been removed, and rename command has been added. I think we should ditch the backtics. I don't mind how the plaintext looks. This should be fine :)

Reply:
0257374 keeps the pipes, however I realized why I used the backticks to begin with because markdown treats things inside `<>` as html, so those had to be escaped with backslashes as well